marketing/models.py

```diff
@@ -0,0 +1,146 @@
+        raise ValidationError(
+            "Configuring follow up email after {} days is not allowed.".format(
+                settings.MARKETING_DELETE_FOLLOWUP_EMAILS_AFTER_DAYS
+            )
+        )
+
+
+class EmailTemplate(models.Model):
+    """
+    Model to store the Email templates
+    """
+    name = models.CharField(
+        unique=True,
+        max_length=100,
+        help_text="Name of the email template. It can be used as an identifier for emails. Must be unique."
+    )
+    subject = models.CharField(
+        max_length=256,
+        help_text="Template for the subject of email"
+    )
+    html_body = models.TextField(
+        help_text="Template for the html body of the email."
+    )
+    plaintext_body = models.TextField(
+        help_text="Template for the text body of the email."
+    )
+    is_active = models.BooleanField(
+        default=False,
+        help_text="If the email is active or not. Emails will be sent for active templates only."
+    )
+    send_after_days = models.IntegerField(
+        validators=[
+            MinValueValidator(1),
+            validate_send_after_days_max_value
+        ],
+        help_text="Number of days after instance activation the email will be sent"
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        """
+        String representation
+        """
+        return self.name
+
+    @cached_property
+    def subject_template(self):
+        """
+        Get Template instance for subject.
+        """
+        return Template(self.subject)
+
+    @cached_property
+    def html_body_template(self):
+        """
+        Get Template instance for html_body.
+        """
+        return Template(self.html_body)
+
+    @cached_property
+    def plaintext_body_template(self):
+        """
+        Get Template instance for plaintext_body.
+        """
+        return Template(self.plaintext_body)
+
+
+class SentEmail(models.Model):
+    """
+    Model to store sent followup emails.
+    """
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+    template = models.ForeignKey(EmailTemplate, on_delete=models.CASCADE)
+    email_subject = models.TextField()
+    email_html_body = models.TextField()
+    email_plaintext_body = models.TextField()
+    sent_at = models.DateTimeField(default=now)
+
+    def __str__(self):
+        """
+        String representation
+        """
+        return "{recipient}<EmailTemplate={template}>".format(
+            recipient=self.user_id,
+            template=str(self.template)
+        )
+
+
+class Subscriber(models.Model):
+    """
+    Model to store followup email subscribers
+    """
+    user = models.OneToOneField(get_user_model(), on_delete=models.CASCADE)
+    receive_followup = models.BooleanField(
+        default=True,
+        help_text="Should the user recieve followup emails?"
```

Comment:
@arjunsinghy96, nit:

```suggestion
        help_text="Should the user receive followup emails?"
```
and to be updated in the corresponding migration as well.